app/models/spree/payment_method.rb

```diff
@@ -70,14 +68,6 @@ def payment_source_class
       raise 'You must implement payment_source_class method for this gateway.'
     end
 
-    def self.available(display_on = 'both')
```

Comment:
I think that this is still used, for example:

https://github.com/openfoodfoundation/openfoodnetwork/blob/49a60374e6f4c48202b1899b09b7c4a42fa0a027/app/models/enterprise.rb#L111

https://github.com/openfoodfoundation/openfoodnetwork/blob/49a60374e6f4c48202b1899b09b7c4a42fa0a027/app/models/enterprise.rb#L343

```
$ git grep  "\.available\>" -- app/helpers app/models lib
app/helpers/enterprises_helper.rb:    payment_methods = current_distributor.payment_methods.available(:front_end).all
app/helpers/order_cycles_helper.rb:      if options[:shipping_and_payment_methods] && (e.shipping_methods.empty? || e.payment_methods.available.empty?)
app/helpers/order_cycles_helper.rb:        if e.shipping_methods.empty? && e.payment_methods.available.empty?
app/helpers/order_cycles_helper.rb:        elsif e.payment_methods.available.empty?
app/models/calculator/flexi_rate.rb:    def self.available?(_object)
app/models/enterprise.rb:      merge(Spree::PaymentMethod.available).
app/models/enterprise.rb:    shipping_methods.any? && payment_methods.available.any?
lib/open_food_network/enterprise_issue_validator.rb:      @enterprise.payment_methods.available.any?
```